src/commands/api/get.js

```diff
@@ -12,11 +12,14 @@ class GetAPICommand extends BaseCommand {
   }
 
   logApiDefinition(response) {
-    const definition = getResponseContent(response)
+    const responseContent = getResponseContent(response)
+    if (responseContent instanceof Promise) {
```

Comment:
Could be better ways of doing this, you can at the moment though create a domain with nothing in it and we will log Promise errors to the user.

Reply:
Yeah I didn't like that it had two different response types. I'll change the `getResponseContent` to what you suggested